account_banking_sepa_direct_debit/models/account_payment_order.py

```diff
@@ -69,6 +68,8 @@ def generate_payment_file(self):
             transactions_count_a += 1
             priority = line.priority
             categ_purpose = line.category_purpose
+            line.compute_sepa()
```

Comment:
This is suspicious: you should normally not need to call compute methods manually.